src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowCreateSpec.groovy

```diff
@@ -408,7 +409,7 @@ source: switchId="${flow.sharedEndpoint.switchId}" port=${flow.sharedEndpoint.po
         }
     }
 
-    @Tags([TOPOLOGY_DEPENDENT])
+    @Tags([HARDWARE])
```

Comment:
If I am not mistaken, we can set up LAG feature on virtual env and this test can be executed (but it seems like only manual setup).